// react-grab/dist/index.cjs
'use strict';

var chunk46DE32OJ_cjs = require('./chunk-46DE32OJ.cjs');

/**
 * @license MIT
 *
 * Copyright (c) 2025 Aiden Bai
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

// src/index.ts
var globalApi = null;
var getGlobalApi = () => {
  return globalApi;
};
if (typeof window !== "undefined") {
  globalApi = chunk46DE32OJ_cjs.init();
}

Object.defineProperty(exports, "DEFAULT_THEME", {
  enumerable: true,
  get: function () { return chunk46DE32OJ_cjs.DEFAULT_THEME; }
});
Object.defineProperty(exports, "formatStack", {
  enumerable: true,
  get: function () { return chunk46DE32OJ_cjs.formatStack; }
});
Object.defineProperty(exports, "getHTMLPreview", {
  enumerable: true,
  get: function () { return chunk46DE32OJ_cjs.getHTMLPreview; }
});
Object.defineProperty(exports, "getNearestComponentName", {
  enumerable: true,
  get: function () { return chunk46DE32OJ_cjs.getNearestComponentName; }
});
Object.defineProperty(exports, "getStack", {
  enumerable: true,
  get: function () { return chunk46DE32OJ_cjs.getStack; }
});
Object.defineProperty(exports, "init", {
  enumerable: true,
  get: function () { return chunk46DE32OJ_cjs.init; }
});
Object.defineProperty(exports, "isInstrumentationActive", {
  enumerable: true,
  get: function () { return chunk46DE32OJ_cjs.Ee; }
});
exports.getGlobalApi = getGlobalApi;
